src/Files.App/Services/WindowsJumpListService.cs

```diff
@@ -1,17 +1,33 @@
 // Copyright (c) 2024 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using Microsoft.Extensions.Logging;
 using System.IO;
 using Windows.Storage;
 using Windows.UI.StartScreen;
 
 namespace Files.App.Services
 {
-	public sealed class JumpListService : IJumpListService
+	public sealed class WindowsJumpListService : IWindowsJumpListService
 	{
 		private const string JumpListRecentGroupHeader = "ms-resource:///Resources/JumpListRecentGroupHeader";
 		private const string JumpListPinnedGroupHeader = "ms-resource:///Resources/JumpListPinnedGroupHeader";
 
+		public async Task InitializeAsync()
+		{
+			try
+			{
+				App.QuickAccessManager.UpdateQuickAccessWidget -= UpdateQuickAccessWidgetAsync;
+				App.QuickAccessManager.UpdateQuickAccessWidget += UpdateQuickAccessWidgetAsync;
```

Comment:
I don't think this belongs in the jump list service, this is related to quick access.

Reply:
This provides user to refresh jump list when user edits quick access widget pinned items.
I'll make it more appropriate event method name.

Reply:
I see, thanks for the clarification.